docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Exporting
 
 ## Overview
 
+Once your data is cleaned, you will need to get it out of OpenRefine and into the system of your choice. OpenRefine outputs a number of file formats, can upload your data directly into Google Sheets, and can export directly to Wikidata. 
+
```

Comment:
Would "use it to create or update Wikidata items" be a better phrasing than "export directly to Wikidata"?